assets/js/modules/idea-hub/components/dashboard/DashboardCTA/index.js

```diff
@@ -0,0 +1,98 @@
+/**
```

Comment:
One thing I missed here 😬  the ACs call for a `assets/js/modules/idea-hub/components/dashboard/DashboardCTA.js` component but this uses a directory. Currently we only use this for components which are composed of sub-components which are local to the larger one. Let's update this to use the path as described in the ACs for consistency.

Reply:
Oops, yes. Good catch.